docs/preview/03-Guidance/use-with-dotnet-and-aspnetcore.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Using Arcus & Serilog in ASP.NET Core"
+layout: default
+---
+
+# Using Arcus & Serilog in ASP.NET Core
+
+When using Arcus Observability & Serilog it is mandatory that your application is configured correctly to ensure the telemetry is correctly send to Azure Application Insights.
+
+## Setting up Serilog with ASP.NET Core
+
+Using Serilog with ASP.NET Core requires you to install an ASP.NET Core specific Serilog package:
+
+```shell
+PM > Install-Package -Name Serilog.AspNetCore
+```
```

Comment:
No, this package isn't implicitly added bc we take a dependency on Serilog functionality and not on ASP.NET Core functionality explicitly; so it could be added to other forms as well (think about Azure Functions).

Reply:
So, I'll resolve this.